src/boot_O2_g3/idle.c

```diff
@@ -1,5 +1,5 @@
-#include "prevent_bss_reordering.h"
 #include "global.h"
+#include "prevent_bss_reordering.h"
```

Comment:
Return back above?

Reply:
Sadly i had to put it after global.h to match idle.c bss

Reply:
This should be fixed (hopefully), along with the problems you are having with the fault PR, in #1292

Reply:
Yeah, I'm looking forward to that PR